examples/asr/speech_to_text.py

```diff
@@ -35,12 +35,21 @@
 
 Hydra logs will be found in "$(./outputs/$(date +"%y-%m-%d")/$(date +"%H-%M-%S")/.hydra)"
 PTL logs will be found in "$(./outputs/$(date +"%y-%m-%d")/$(date +"%H-%M-%S")/lightning_logs)"
+
+ python speech_to_text.py AudioToTextDataLayer.manifest_filepath="./an4/train_manifest.json" AudioToTextDataLayer_eval.manifest_filepath="./an4/test_manifest.json" hydra.run.dir="." pl.trainer.gpus=2 pl.trainer.max_epochs=100
+
+Override some args of optimizer
+ python speech_to_text.py AudioToTextDataLayer.manifest_filepath="./an4/train_manifest.json" AudioToTextDataLayer_eval.manifest_filepath="./an4/test_manifest.json" hydra.run.dir="." pl.trainer.gpus=2 pl.trainer.max_epochs=2 optim.args.params.betas=[0.9,0.22] optim.args.params.weight_decay=0.0000005
+
+Overide optimizer entirely
+ python speech_to_text.py AudioToTextDataLayer.manifest_filepath="./an4/train_manifest.json" AudioToTextDataLayer_eval.manifest_filepath="./an4/test_manifest.json" hydra.run.dir="." pl.trainer.gpus=2 pl.trainer.max_epochs=2 optim.name=adamw optim.lr=0.001 ~optim.args +optim.args.betas=[0.9,0.22] +optim.args.weight_decay=0.0000005
+
 """
 
 
 @hydra.main(config_path="conf", config_name="config")
```

Comment:
My main question is: Suppose under "conf" I have: "conf/qn15x5.yaml and conf/qn5x3.yaml".
I understand that I can set config_name param of @hydra.main decorator to point to either qn15x5 or qn5x3. But is there a way to specify this via command line to speech2text.py script?

Reply:
@okuchaiev this is config groups you can do them with .yaml's or structured config: https://hydra.cc/docs/next/tutorials/basic/your_first_app/config_groups, https://hydra.cc/docs/next/tutorials/structured_config/config_groups

This is exactly how we'll support multiple language, architecture's, etc.